Tools/autotest/sim_vehicle.py

```diff
@@ -851,6 +852,11 @@ def start_mavproxy(opts, stuff):
     location = find_location_by_name(find_autotest_dir(), cmd_opts.location)
     progress("Starting up at %s (%s)" % (location, cmd_opts.location))
 
+if cmd_opts.use_dir is not None:
+    new_dir = cmd_opts.use_dir
+    os.makedirs(new_dir)
```

Comment:
@peterbarker The reason I said to check for existence first is that it will also give an exception if it can't create the directory - and in the first case we should just proceed but in the second we should exit with an error.